hpcgap/lib/oper.g

```diff
@@ -1276,6 +1276,78 @@ BIND_GLOBAL( "DeclareAttribute", function ( arg )
 end );
 
 
+##############################################################################
+##
+##  This is a temporary (I hope!) hack for HPCGAP, which introduces thread-local
```

Comment:
Leaving an "I hope!" in a comment is problematic -- somebody who reads this code later on may not know who the "I" refers to.

Moreover, I am missing in this comment (and also in the text of the pull request) an actual explanation of the problem these are meant to solve. Personally, I'd write something more in the spirit of this:
>  Thread local attributes.
>
>  Thread local attributes are similar to regular attributes. The difference comes in storage:
>  Regular attributes are stored (if they are stored at all) globally, i.e., accessible by all thread,
>  while thread local attributes are stored (as the name implies) separately for each thread.
>
Then explain why this might be useful; but also explain other consequences and potential pitfalls. As to the "hopefully this is temporary": First off, if we offer this, and expect people to use it, I am not sure how it can be "temporary" at all (so I guess I am asking here: what is the expected endgame? When would these be removed, and what might replace them???).  So perhaps we should not say that at all. But if we say it, then we should explain that (i.e. why they are temporary, and what that *means* -- does it e.g. mean that use of them should be avoided? But then, who should use them? etc.)